regression-tests.recursor-dnssec/test_Protobuf.py

```diff
@@ -931,7 +933,7 @@ def testA(self):
         # then the response
         msg = self.getFirstProtobufMessage()
         self.checkProtobufResponse(msg, dnsmessage_pb2.PBDNSMessage.UDP, res)
-        self.checkProtobufPolicy(msg, dnsmessage_pb2.PBDNSMessage.PolicyType.QNAME, 'zone.rpz.', '*.test.example.', 'sub.test.example')
+        self.checkProtobufPolicy(msg, dnsmessage_pb2.PBDNSMessage.PolicyType.QNAME, 'zone.rpz.', '*.test.example.', 'sub.test.example', dnsmessage_pb2.PBDNSMessage.PolicyKind.NoAction)
```

Comment:
Not your fault at all but I'm a bit sad we only have `NoAction` examples here.. :-/

Reply:
Yes, will make a note to add some more one day.